seatunnel-connectors-v2/connector-elasticsearch/src/test/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/serialize/ElasticsearchRowSerializerTest.java

```diff
@@ -71,8 +71,6 @@ public void testSerializeUpsert() {
                         + "\"} }\n"
                         + "{ \"doc\" :{\"name\":\""
                         + name
-                        + "\",\"id\":\""
-                        + id
```

Comment:
why delete this?

Reply:
Because the primary_keys field has already been assigned an id, there is no need to specifically indicate the id in the document

Reply:
Why can't the code of other connectors that haven't been modified pass unit test

Reply:
please re-trigger failed ci.

> Why can't the code of other connectors that haven't been modified pass unit test